django/http/response.py

```diff
@@ -57,6 +58,7 @@ def __init__(self, content_type=None, status=None, reason=None, charset=None):
                 raise ValueError('HTTP status code must be an integer from 100 to 599.')
         self._reason_phrase = reason
         self._charset = charset
+        self._default_content_type = content_type is None
```

Comment:
This should live in FileResponse.__init__() checking if content_type was not None in `kwargs`. 

The variable name should either be something like `use_default_...` or `guess_default_...` or else `explicit_..._set` or similar.

Reply:
Is the lack of preceding underscore on purpose?
Also i don't agree with the idea of `use_default_content_type` or `guess_default_content_type` - after all, it's used to inform `set_headers()` to NOT use the default content_type value.
`explicit_content_type_set` is better, but it requires double negatives:
`self.explicit_content_type_set = content_type is not None`
and later
`if not self.explicit_content_type_set: [guess content type]`

So maybe just change it to `_default_content_type_set`?

Reply:
Sounds good. (The lack of a `_` wasn't central to the point I was trying to communicate.)